app/ts/utils/etherscan.tsx

```diff
@@ -0,0 +1,13 @@
+import { Signal, useComputed } from '@preact/signals'
+import { AccountAddress } from '../types/types.js'
+
+interface EtherScanAddressProps {
+	name: string,
+	address: Signal<AccountAddress> | undefined
+}
+
+export const EtherScanAddress = ({ address, name }: EtherScanAddressProps) => {
+	if (address === undefined) return '?'
+	const etherScan = useComputed(() => `https://etherscan.io/address/${ address.value }`)
+	return <a target = '_blank' rel = 'noopener noreferrer' href = { etherScan }>{ name }</a>
```

Comment:
Should we put one of those new-window icons after `{ name }`?

Reply:
whats that?

Reply:
https://fontawesome.com/v4/icon/external-link
https://ux.stackexchange.com/questions/107823/is-there-a-common-or-typical-icon-for-an-external-link-to-a-company-web-site

Reply:
added